### PR TITLE
Barcodes in own schema

### PR DESCRIPTION
**Pull request by squirrelo**
Depends on #429

This moves the barcode table and associated tables to their own schema, keeping the foreign keys from the ag schema.

### CODE REVIEW

**Comment by wasade** [comment]
Was registering kits and verifying barcodes tested? Was results pull down tested?

**Comment by squirrelo** [comment]
first two were, pull down no, since that is not touched by this.

File: amgut/db/initialize.sql [hunk 1]
```diff
@@ -1,4 +1,4 @@
-SET search_path TO ag, public;
+SET search_path TO ag, public, barcodes;
```
Comment (adamrp) [inline amgut/db/initialize.sql]: probably best to put `barcodes` before public; these namespaces are searched in the order specified

File: amgut/lib/data_access/ag_data_access.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def __init__(self, con=None):
         else:
             self.connection = con
         cur = self.get_cursor()
-        cur.execute('set search_path to ag, public')
+        cur.execute('set search_path to ag, public, barcodes')
```
Comment (adamrp) [inline amgut/lib/data_access/ag_data_access.py]: ordering of items in search path

File: amgut/lib/data_access/sql_connection.py [hunk 3]
```diff
@@ -23,7 +23,7 @@ def __init__(self, con=None):
         else:
             self._connection = con
 
-        self.execute('SET search_path TO ag, public')
+        self.execute('SET search_path TO ag, public, barcodes')
```
Comment (adamrp) [inline amgut/lib/data_access/sql_connection.py]: ordering

**Comment by adamrp** [comment]
Had the same relatively minor comment 3 times. :+1: after that's addressed and #429 is merged.

**Comment by squirrelo** [comment]
changed ordering for search_path, ready to go.

**Comment by wasade** [comment]
@squirrelo, conflicts

**Comment by squirrelo** [comment]
resolved

**Pull request merged**